src/OV/OpenVarioMenu.cpp

```diff
@@ -207,16 +207,6 @@ SystemMenuWidget::Prepare([[maybe_unused]] ContainerWindow &parent,
                      "Update System", argv);
   });
 
-  AddButton("Update Maps", [](){
-    static constexpr const char *argv[] = {
-      "/usr/bin/update-maps.sh", nullptr
-    };
-
-    RunProcessDialog(UIGlobals::GetMainWindow(),
-                     UIGlobals::GetDialogLook(),
-                     "Update Maps", argv);
-  });
```

Comment:
Wait, this removes a feature. This is confusing. If it belongs somewhere else, then this commit should move it, and not delete it.

Reply:
I reverted two commits and make a new one. Now submenu "Update Maps" is moved to the submenu "Files"